deploy/L2TokenGatewaySpell.sol

```diff
@@ -38,11 +41,25 @@ contract L2TokenGatewaySpell {
     function rely(address usr) external { l2Gateway.rely(usr); }
     function deny(address usr) external { l2Gateway.deny(usr); }
     function close() external { l2Gateway.close(); }
+
+    function registerToken(address l1Token, address l2Token) public { 
+        l2Gateway.registerToken(l1Token, l2Token);
+        AuthLike(l2Token).rely(address(l2Gateway));
+    }
```

Comment:
I guess the idea of extracting this as its specific function with also external visibility is to being able to use it directly in a future spell, right? Shouldn't be better to keep this function as it was receiving arrays?

Reply:
Yes exactly. 
Yeah I think it's indeed better to move the loop to `registerTokens`. Done.